preprocess.py

```diff
@@ -212,14 +251,15 @@ def workflow(start=None, stop=None):
                 "AWS_ACCESS_KEY_ID": os.environ["AWS_ACCESS_KEY_ID"],
                 "AWS_SECRET_ACCESS_KEY": os.environ["AWS_SECRET_ACCESS_KEY"],
             })
+            cluster.adapt(minimum=1, maximum=100)
         with cluster.get_client() as client:
-            client.restart()
             futures = client.map(process_file, filenames)
             futures = client.map(write_delta, futures, retries=10)
             wait(futures)
             tables = ["comment", "pr", "commit", "create", "watch", "fork"]
             futures = client.map(compact, tables)
             wait(futures)
+            save_results()
```

Comment:
I'm going to say some stuff that I think you already agree, but I want to make explicit just to make sure that we're on the same page.

I don't think that these really classify as "results" in the same way as we had in TPCH.  These are really specific to one repo (dask) and I think that we probably want to set things up to handle lots of different situations.  Probably we want to find ways to transform our data such that we can ask lots of interesting questions quickly in a streamlit app. 

Also, I'd prefer if we stopped calling these "results" and started using more specific language like `repos_mentioning_dask` or something.  This makes more space in the future I think.

But, I think that you're just doing this in a quick and dirty way so that we can have *something*.  If this is the case then feel free to merge, although I wouldn't mind ...

1.  adding a comment to this effect so that others don't think that we think this is a good state
2. rename away from `results`

Reply:
Yeah, we're on the same page. I'll push up a rename + comment and merge